src/pull-request-table/PullRequestRowTimeline.tsx

```diff
@@ -0,0 +1,138 @@
+import TimelineCommit from '../timeline/TimelineCommit';
+import MuiTimeline from '@mui/lab/Timeline';
+import { format } from 'date-fns';
+import {
+  TimelineConnector,
+  TimelineContent,
+  TimelineDot,
+  TimelineItem,
+  timelineOppositeContentClasses,
+  TimelineOppositeContent,
+  TimelineSeparator,
+} from '@mui/lab';
+import { Typography } from '@mui/material';
+import SaveAsIcon from '@mui/icons-material/SaveAs';
+import TimelineMerged from '../timeline/TimelineMerged';
+import TimelineClosed from '../timeline/TimelineClosed';
+import TimelineComment from '../timeline/TimelineComment';
+import TimelineReview from '../timeline/TimelineReview';
+import TimelineLabeled from '../timeline/TimelineLabeled';
+import TimelineReopened from '../timeline/TimelineReopened';
+
+export default function PullRequestRowTimeline({ pullRequest }: any) {
```

Comment:
If I'm reading this right, shouldn't this be `PullRequestKeyMetrics`.

Reply:
Good catch resolved in  1710cc0ef2d39c5f85891ceb89aec23e1115e93f